Block/Adminhtml/Form/Field/Accounts.php

```diff
@@ -0,0 +1,199 @@
+     * @throws LocalizedException
+     */
+    private function getBankRenderer()
+    {
+        if (!$this->bankRenderer) {
+            $this->bankRenderer = $this->getLayout()->createBlock(
+                BankColumn::class,
+                '',
+                ['data' => ['is_render_to_js_template' => true]]
+            );
+        }
+        return $this->bankRenderer;
+    }
+
+    /**
+     * @param \Magento\Framework\Data\Form\Element\AbstractElement $element
+     * @return string
+     */
+    protected function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractElement $element)
+    {
+        $html = parent::_getElementHtml($element);
+
+        $bankList = $this->json->serialize($this->getBanks());
+
+        $script = '<script type="text/javascript">
+                require(["jquery"], function ($) {
+                    $(document).ready(function($) {
+                        var banList = '.$bankList.';
+                        $(".country-select").each(function(elem){
+                            var selected = $(this).val();
+                            var bankElem = $(this).parent().parent("tr").find(".bank-select");
+                            if(selected){
+                                selectedBank = bankElem.val();
+                                bankElem.empty();
+                                if(banList[selected].length){
+                                    $.each(banList[selected], function (i, p) {
+                                        bankElem.append($("<option></option>").val(p.code).html(p.title));
+                                    });
+
+                                    bankElem.val(selectedBank);
+                                }
+                            }
+                        });
+
+                        $(document).on("change", ".country-select", function() {
+                            var activeCountry = this.value;
+                            var bankElem = $(this).parent().parent("tr").find(".bank-select");
+                            bankElem.empty();
+
+                            if(banList[activeCountry].length){
+                                $.each(banList[activeCountry], function (i, p) {
+                                    bankElem.append($("<option></option>").val(p.code).html(p.title));
+                                });
+
+                                bankElem.prop("disabled", false);
+                            }
+                        });
+
+                        $(document).on("click", "#payment_us_kevin_payment_additional_bank_additional_bank_list .action-add", function() {
+                            $(this).closest("table").find("tr:last").find(".bank-select").prop("disabled", true);
+                        });
+                    });
+                });
+            </script>';
+        $html .= $script;
+        return $html;
+    }
+
+    /**
+     * @return array
+     */
+    protected function getBanks(){
```

Comment:
Can we stick to the same code conventions and have the first curly bracket on the new line? And spaces after every `if`, `for` and other keywoards? It just get my eye sometimes.